cfgov/jinja2/v1/events/_macros.html

```diff
@@ -291,3 +316,81 @@ <h2 class="event-venue_heading">{{ city }}</h2>
    {%- endfor -%}
 {%- endfor -%}
 {% endmacro %}
+
+{# ==========================================================================
+
+   event_media()
+
+   ==========================================================================
+
+   Description:
+
+   Create event archive media markup when given:
+
+   event: An event from a query result.
+
+   ========================================================================== #}
+
+{% macro event_media( event ) %}
+{% set col_classes = 'content-l_col
+                      content-l_col-1-2
+                      block
+                      block__padded-top
+                      block__flush-top
+                      block__flush-bottom'
+%}
+
+{% if event.speech_transcript or event.video_transcript or event.flickr_url %}
+<div class="block block__border-top">
+    <div class="event-archive_media-container
+                content-l
+                content-l__large-gutters">
+        {% if event.flickr_url %}
+        <section class="{{ col_classes }} block__padded-bottom">
+            <h4>Photography</h4>
+            <p class="u-flexible-container">
+                <img class="u-flexible-container_inner"
+                     src='{{ event.flickr_url }}' >
+            </p>
+            <p class="speech_link-container">
+                <a class="jump-link
+                          jump-link__external-link
+                          jump-link__underline"
+                    href="{{ event.flickr_url }}">
+                    See the album for the Event
+                </a>
+            </p>
+        </section>
+        {% endif %}
+        <div class="{{ col_classes }}">
+            {% if event.speech_transcript %}
+            <section class="block block__flush-top">
+                <h4>Speech text</h4>
+                <p>
+                    <a href="{{ event.speech_transcript }}"
+                       class="jump-link
+                              jump-link__underline">
+                       Download speech
+                       <span class="cf-icon cf-icon-pdf"></span>
+                    </a>
+                </p>
+            </section>
+            {% endif %}
+            {% if event.video_transcript %}
+            <section class="block block__flush-bottom">
+                <h4>Video transcript</h4>
+                <p>
+                    <a href="{{ event.video_transcript }}"
+                       class="jump-link
+                              jump-link__underline">
+                       Download transcript
+                       <span class="cf-icon cf-icon-pdf"></span>
```

Comment:
Should this just be the generic download icon? Are we using the pdf download anymore? See https://github.com/cfpb/cfgov-refresh/pull/1573